odoo/tools/translate.py

```diff
@@ -151,8 +151,19 @@ def encode(s):
     'aria-keyshortcuts', 'aria-placeholder', 'aria-roledescription', 'aria-valuetext',
     'value_label',
 }, lambda e: True)
+
+def translate_attrib_value(node):
+    # check if the value attribute of a node must be translated
+    classes = node.attrib.get('class', '').split(' ')
+    return (
+        (node.tag == 'input' and node.attrib.get('type', 'text') == 'text')
+        and 'datetimepicker-input' not in classes
+        or (node.tag == 'input' and node.attrib.get('type') == 'hidden')
+        and 'o_translatable_input_hidden' in classes
```

Comment:
Why make the classes and cases so specific, and continuing to stack them up? How would developers be aware of those incredibly specific rules, and the fact that they can suddenly use a magical class `o_translatable_input_hidden`?

Is there no way to have a few generic classes instead (`o_translate` and `o_notranslate`?), that could be used to override the default behavior for any element? @mart-e ?

How do we plan to document this?